ReSharper.FSharp/src/FSharp/FSharp.Psi.Services/src/Util/FcsTypeUtil.fs

```diff
@@ -19,3 +25,7 @@ let getFunctionTypeArgs includeReturnType fcsType =
                 acc
 
     loop fcsType [] |> List.rev
+
+let stripNull (fcsType: FSharpType) =
+    if fcsType.HasNullAnnotation then fcsType.TypeDefinition.AsType()
```

Comment:
It seems this function does much more than stripping the nullability. Does it keep the type arguments properly? If not, let's not add such an API, it's too easy to misuse it.

Reply:
This function has been removed from the API

Reply:
I've added a test for the case you provided. 
`fcsType.TypeDefinition.AsType()` here works as expected with type arguments

https://github.com/JetBrains/resharper-fsharp/blob/ber.a/autoImportSpecifyTypes/ReSharper.FSharp/test/data/features/intentions/specifyTypes/Import%20types%20-%20Type%20with%20null%2003%20-%20Generics.fs